tests/nextflow_schema.json

```diff
@@ -0,0 +1,259 @@
+{
```

Comment:
Similar to the question for the `tests/assets/schema_input.json`. Was this put here when trying to get the tests to work? Could you remove this file if so?

Reply:
Yes! Good call.